### ISSUE TITLE
2bn4Digial community keeps asking to join....

### ISSUE THREAD

**Issue by tmazuryk** [Cores-ts/Fundingbox-support#133]
Although I ahem joined 2bn4Digital, it keeps asking me to join....

http://2bn4digital.fundingbox.com

![zrzut ekranu 2016-10-13 o 10 01 44](https://cloud.githubusercontent.com/assets/16816225/19341044/1f3028b0-912c-11e6-9284-ec51f49c4efa.png)

**Comment by xurxo** [comment]
I don't think so. The button in the main page remains even if you are already a member, but if you click you should see a message telling you that you're a member, right?

We'll remove the button in the next release.
Thanks

**Comment by tmazuryk** [comment]
Yes, please remove it... Or we can change it to: "You are already a member. Start exploring."

**Comment by xurxo** [comment]
Event better, thanks

2016-10-14 9:46 GMT+02:00 tmazuryk <notifications@github.com>:

**Comment by tmazuryk** [comment]
When do you plan to fix this?

**Comment by xurxo** [comment]
At the end of the week we'll deploy a new realease with an improved signup process, and also fixing this
